### PR TITLE
Issue #324 - Adding new test for pull showing how to mock a pull wallet

### PR DESCRIPTION
**Pull request by emilianodellacasa**
As requested in issue #324, I have created a test for pull command showing hot to mock a pull request of a wallet.

Basically this is what I am doing:

- Add a mock remote;
- Generating a fake response for a wallet, by creating a proper wallet (in a separate directory);
- stubbing the response of _/wallet_ call;
- execute the _Pull_ command;
- testing if the wallet has been correctly merged;

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#418](https://github.com/zold-io/zold/issues/418) is now in scope, role is `REV`

File: test/commands/test_pull.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Author:: Yegor Bugayenko (yegor256@gmail.com)
+# Copyright:: Copyright (c) 2018 Yegor Bugayenko
+# License:: MIT
+class TestPull < Minitest::Test
+  def test_pull_wallet
+    FakeHome.new.run do |home|
+      remotes = home.remotes
+      remotes.add('localhost', 80)
+      wallet_json = home.create_wallet_json
+
```
Comment (yegor256) [inline test/commands/test_pull.rb]: @emilianodellacasa see https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

File: test/commands/test_pull.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# PUSH test.
+# Author:: Yegor Bugayenko (yegor256@gmail.com)
+# Copyright:: Copyright (c) 2018 Yegor Bugayenko
+# License:: MIT
+class TestPull < Minitest::Test
+  def test_pull_wallet
+    FakeHome.new.run do |home|
+      remotes = home.remotes
+      remotes.add('localhost', 80)
+      wallet_json = home.create_wallet_json
```
Comment (yegor256) [inline test/commands/test_pull.rb]: @emilianodellacasa see https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html Just `json` is enough

File: test/commands/test_pull.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+      wallet_id = Zold::JsonPage.new(wallet_json).to_hash['id']
```
Comment (yegor256) [inline test/commands/test_pull.rb]: @emilianodellacasa just `id` is enough

**Comment by yegor256** [comment]
@emilianodellacasa a few comments above

**Comment by emilianodellacasa** [comment]
@yegor256 Thanks for the comments, all are fixed in my last commit

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zold-io/zold#418` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#418](https://github.com/zold-io/zold/issues/418) is now out of scope

**Pull request merged**